### DISCUSSION TITLE
Can't use discord on work account with no phone number

### DISCUSSION THREAD

**Discussion by amrtgaber** [TanStack/router#2705] [General]
I'm guessing this isn't high priority and I'm not sure how much traffic you all want in the discord but I'm on a work account that has no associated phone number so I can't verify my phone in order to ask questions or participate in the discord. Just wanted to leave this here in case it's something you all wanted to consider. Either way, thanks for tanstack, the docs, and all the help!

**Comment by ukgrim** [comment]
Hey,
The phone verification is a requirement from Discord for added security, and it’s needed for certain accounts (like work ones) to participate fully. Unfortunately, if your account doesn't have a phone number linked, that’s why you're unable to verify it. 

[More here](https://support.discord.com/hc/en-us/articles/6181726888215-How-to-Verify-Your-Discord-Account#h_01G3CTDAACPA8E9BRFWCQNSBY4)

**Reply by amrtgaber** [reply]
No problem, I assumed as much. Thanks.
